### ISSUE TITLE
ERROR(6002): An exception has occured while patching. Report to Nemesis' author immediately

### ISSUE THREAD

**Issue by downqc** [ShikyoKira/Project-New-Reign---Nemesis-Main#608]
![gggggggggggggggggg](https://user-images.githubusercontent.com/91120932/134140849-d4e5e92d-d343-4f36-8756-188e4266e253.PNG)

**Comment by downqc** [comment]
i cant update my nemesis and i have reinstal nemesis so my game animation dont work

**Comment by MagicalPrincessUnicorn** [comment]
Please try the solution I suggested in #591 

Seems to work for most people.

**Comment by downqc** [comment]
yeah ty i have see after post all the other post for the same error and
have fix it with the solution you gived, it worked fine when disable the
real time protection

ty for your time and your answer i appreciated.
